<?xml version="1.0"?>
<launch>

    <!-- Republishes global planner pose array msg to nav_msg path -->
    <arg name="global_plan_topic" default="/move_base/GlobalPlannerWithOrientations/plan" />
    <arg name="path_topic" default="/move_base/GlobalPlannerWithOrientations/path_plan" />

    <node pkg="mcr_navigation_tools" type="pose_array_to_path_node" name="pose_array_to_path" output="screen" ns="mcr_navigation_tools" >
        <param name="node_frequency" value="10.0" />
        <remap from="~pose_array" to="$(arg global_plan_topic)"/>
        <remap from="~path" to="$(arg path_topic)"/>
    </node>

</launch>
